.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: python
 python:
-  - "2.7"
   - "3.5"
```

Comment:
This gives us a "Python 3.5" environment, which by default uses prod artefacts, so we can actually remove this entry under `matrix` below:
```yaml
     - name: "prod artefacts"
      env: STAGE="prod"
```